<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="vlAllContent">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
    <item>
     <layout class="QHBoxLayout" name="hlTopBar">
      <item>
       <spacer name="hsForTopBar">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbConnect">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlMainContent" stretch="1,5">
      <item>
       <widget class="QListWidget" name="lwTabs">
        <item>
         <property name="text">
          <string>Welcome</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Settings</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monitor</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="swContent">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="pWelcome">
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>49</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Welcome</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="pSettings">
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>49</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="pMonitor">
         <widget class="QTextEdit" name="teMonitor">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>601</width>
            <height>281</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="leCLI">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>310</y>
            <width>511</width>
            <height>24</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pbSend">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>310</y>
            <width>80</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lwTabs</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>swContent</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
